app/models/hackbot/helpers.rb

```diff
@@ -0,0 +1,30 @@
+module Hackbot
+  module Helpers
+    extend ActiveSupport::Concern
+
+    def access_token
+      team.bot_access_token
+    end
+
+    def msg
+      return nil unless event[:type] == 'message'
```

Comment:
This is explicit. What if Slack adds another event type that sets `event[:text]`?

This code makes it super clear that it returns nil unless it's a message. Also this code isn't new, this was copied over from `interaction.rb`.